Update sec_advisors.tex

cedreux opened the pull request:
Correct C_loss expression and use C_beta_% instead of C_beta.

cedreux committed:
Update sec_advisors.tex

Correct C_loss expression and use C_beta_% instead of C_beta.

jhillairet commented on sec_advisors.tex:
```diff
@@ -630,7 +631,7 @@ \section{Plasma Energy}
 \hat W = 2\pi^2 C_{loss}  \varepsilon^2  \hat n \hat T R^3
 \label{eq:total_energy}
 \end{equation}
-where $C_{loss} = 3 \times 10^{19} \times 10^{-3} e$.
+where $C_{loss} = 6 \pi^2 \times 10^{19} \times 10^{-3} e$.
```
si on rajoute le 2pi^2 dans l'expression de C_loss, alors il faut l'enlever de l'expression de W (juste au dessus), non ?

cedreux replied:
Effectivement, tu as raison. Je vais voir si je peux modifier ça.

cedreux committed:
Update sec_advisors.tex

Remove one useless reference to C_loss, remove 2pi**2 from the plasma energy reformulation.

jhillairet commented:
super, merci. Je merge et je mets à jour overleaf pour intégrer ces modifications.
